tests/PHPUnit/Framework/TestCase/UnitTestCase.php

```diff
@@ -18,16 +21,45 @@
  */
 abstract class UnitTestCase extends \PHPUnit_Framework_TestCase
```

Comment:
Some tests in the Unit/ directory do require the environment (probably because they depend on singletons or something). We could make them integration test cases, but I think that's unrelated to this PR.

I could change the name of the class to PiwikTestCase or something so it doesn't mark a test case as "unit".